frontend/jest.config.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    transform: {
+        '^.+\\.tsx?$': 'babel-jest',
+    },
+    testEnvironment: 'node',
```

Comment:
I think this might need to be `testEnvironment: 'jsdom'`

Not sure if it will fix your error though

https://jestjs.io/docs/configuration

Reply:
Thanks, i'll try